src/gameObjects/player.js

```diff
@@ -0,0 +1,36 @@
+export default class Player {
+  constructor(scene, x, y) {
+    this.scene = scene;
+    this.init(x, y);
+  }
+
+  init(x, y) {
+    this.sprite = this.scene.matter.add.sprite(0, 0, "player", 0);
+
+    const { Body, Bodies } = Phaser.Physics.Matter.Matter;
+    const { width: w, height: h } = this.sprite;
+    const mainBody = Bodies.rectangle(0, 0, w * 0.6, h, {
+      chamfer: { radius: 10 },
```

Comment:
**suggestion**: Consider making the chamfer radius a configurable parameter to allow for more flexible character design.